Formula/homeshick.rb

```diff
@@ -0,0 +1,122 @@
+class Homeshick < Formula
+  desc "Git dotfiles synchronizer written in bash"
+  homepage "https://github.com/andsens/homeshick"
+  url "https://github.com/andsens/homeshick/archive/1.0.0.tar.gz"
+  sha256 "8bd3c46f1cfd68d82d97fa72a68a07c966092c77f276d1335cb390b2ec6062bf"
+  head "https://github.com/andsens/homeshick.git"
+
+  option "with-fish", "Build fish bindings"
+  option "with-csh", "Build csh bindings"
+  option "without-bash", "Build without bash bindings"
+
+  if (build.without? "bash") && (build.without? "fish") && (build.without? "csh")
+    odie <<-EOS.undent
+      You need to enable at least one of `--with-csh` or `--with-fish` or
+      omit `--without-bash` for a functional homeshick installation.
+    EOS
+  end
+
+  def install
+    prefix.install "lib"
+    prefix.install "bin/homeshick"
+    if build.with? "bash"
+      prefix.install "homeshick.sh"
+    end
+    if build.with? "fish"
+      prefix.install "homeshick.fish"
+    end
+    if build.with? "csh"
+      prefix.install "bin/homeshick.csh"
+    end
+    zsh_completion.install "completions/_homeshick"
+    bash_completion.install "completions/homeshick-completion.bash"
+    if build.head? && (build.with? "fish")
+      fish_completion.install "completions/homeshick.fish"
+    end
+  end
+
+  patch :DATA
+
+  def caveats
+    s = <<-EOS.undent
+      `export HOMESHICK_DIR=/usr/local/opt/homeshick`
```

Comment:
Homebrew won't always be in `/usr/local`. Should be using the `opt_prefix` style helpers.